app/components/ui/atoms/PersonPod.js

```diff
@@ -88,31 +98,40 @@ const PersonText = ({
   isStatusShown = false,
   status = '',
   ...props
-}) => (
-  <Box {...props} m={2}>
-    <RegularP>{name}</RegularP>
-    <RegularP>{institution}</RegularP>
-    {isKeywordClickable &&
-      keywords &&
-      keywords.map(keyword => (
-        <SmallAction
-          data-test-id="clickable-keyword"
-          key={keyword}
-          onClick={() => onKeywordClick(keyword)}
-        >
-          {keyword}
-        </SmallAction>
-      ))}
-    {!isKeywordClickable &&
-      keywords &&
-      keywords.map(keyword => (
-        <SmallP data-test-id="non-clickable-keyword" key={keyword}>
-          {keyword}
-        </SmallP>
-      ))}
-    {isStatusShown && <SmallP>{status}</SmallP>}
-  </Box>
-)
+}) => {
+  let keywordList
+  if (isKeywordClickable) {
+    keywordList = keywords.map(keyword => (
+      <SmallAction
+        data-test-id="clickable-keyword"
+        key={keyword}
+        onClick={() => onKeywordClick(keyword)}
+      >
+        {keyword}
+      </SmallAction>
+    ))
+  } else {
+    keywordList = keywords.map(keyword => (
+      <span data-test-id="non-clickable-keyword" key={keyword}>
+        {keyword}
+      </span>
+    ))
+  }
+
+  const separatedKeywords = keywordList.reduce(
+    (accu, elem) => (accu === null ? [elem] : [...accu, ', ', elem]),
+    null,
+  )
+
+  return (
+    <CollapsibleBox m={2} {...props}>
+      <RegularP>{name}</RegularP>
+      <RegularP>{institution}</RegularP>
+      <SmallP>{separatedKeywords}</SmallP>
```

Comment:
To me, it's not super obvious just by looking at this `return` block, what is actually being returned. Without first looking at what separatedKeywords is & what keywordsList is, I would look at this block and assume that `separatedKeywords` was just a string, sitting inside a paragraph tag. In fact, it's a bunch of `SmallAction`s or `span`s.

I'm wondering if there's a way that we could break this block up into smaller pieces, so that `PersonPodContainer` isn't responsible for all this logic. If we captured each piece of logic in a different component, we could be more explicit on this line.

I'm thinking something like this, so that PersonPodContainer is only responsible for rendering a list of keywords:
```
<CollapsibleBox m={2} {...props}>
      <RegularP>{name}</RegularP>
      <RegularP>{institution}</RegularP>
      <KeywordsList>
     {isStatusShown && <SmallP>{status}</SmallP>}
</CollapsibleBox>
```

Then `KeywordsList` is separate component, responsible for:
1. mapping over `keywords`
1. checking `isKeywordClickable` and rendering either clickable keywords (`<SmallAction>`) or non-clickable keywords (`<p>`)
1. reducing

```
const KeywordsList = (isKeywordClickable, keywords) => {
    keywords
      .map(keyword => isKeywordClickable ? <ClickableKeyword {keyword}> : <NonClickableKeyword {keyword}>)
      .reduce((accu, elem) => (accu === null ? [elem] : [...accu, ', ', elem]), null)
}
```

Then `ClickableKeyword` and `NonClikcableKeyword` are responsible for taking a keyword and rendering a `SmallAction` or `SmallP` respectively & adding the appropriate props.

What do you reckon?